### ISSUE TITLE
bulkio: index backfill progress appears to no longer work as expected

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#55267]
I built an index on a local one-node cluster (version built from `master`) with TPCH, expecting to see index backfill progress in the jobs page. Instead, I didn't see progress updates. I refreshed every second or so for a minute, and here are the only 3 states I observed:

![image](https://user-images.githubusercontent.com/43821/95253620-a6614180-07ec-11eb-9a8f-036cd2141a07.png)
![image](https://user-images.githubusercontent.com/43821/95253626-ab25f580-07ec-11eb-93a4-c5fac67fc16c.png)
![image](https://user-images.githubusercontent.com/43821/95253769-e1fc0b80-07ec-11eb-958c-e7f171cfac7f.png)

Not sure if this is bulk io or schema, but I do think it's likely a release blocker.

cc @lucy-zhang @ajwerner @dt

**Comment by lucy-zhang** [comment]
I wasn't thinking clearly about this earlier when I looked at this. I agree this is a release blocker, but does that mean we shouldn't start an RC until this is fixed?

**Comment by jordanlewis** [comment]
Seems fine to start an RC without this being fixed.

**Comment by ajwerner** [comment]
I’ll have a look in the morning

**Comment by dt** [comment]
Is this a regression compared to 20.1? I thought backfiller only update progress every 2min since 19.2.

**Comment by jordanlewis** [comment]
That would explain things, because I was testing index backfills that were finished in less than 2 minutes.

**Comment by lucy-zhang** [comment]
Let me just try this with some bigger tables.

**Comment by lucy-zhang** [comment]
<img width="1203" alt="Screen Shot 2020-10-07 at 12 51 04 AM" src="https://user-images.githubusercontent.com/3175520/95288795-4479f980-0837-11eb-8c2e-e8d2915ca046.png">

Backfiller progress seems to still be working. I saw this jump from 0% to 87.5% after around 2-3 minutes. But I also added another index which took 8 minutes to build (with most of that spent in backfill/validation) and it went from 0% to finished with no apparent intermediate progress. Is this what you'd expect?
